nilearn/plotting/img_plotting.py

```diff
@@ -586,7 +611,15 @@ def plot_stat_map(stat_map_img, bg_img=MNI152TEMPLATE, cut_coords=None,
                     np.logical_not(stat_map_data._mask)])
         stat_map_max = np.nanmax(stat_map_data)
         stat_map_min = np.nanmin(stat_map_data)
+
+    if symmetric_cbar == 'auto':
+        symmetric_cbar = stat_map_min < 0
```

Comment:
Ok

Sent from my phone. Please forgive brevity and mis spelling



On Jan 7, 2015, 08:43, at 08:43, bthirion <notifications@github.com> wrote:
>> @@ -586,7 +611,15 @@ def plot_stat_map(stat_map_img,
>bg_img=MNI152TEMPLATE, cut_coords=None,
>>                      np.logical_not(stat_map_data._mask)])
>>          stat_map_max = np.nanmax(stat_map_data)
>>          stat_map_min = np.nanmin(stat_map_data)
>> +
>> +    if symmetric_cbar == 'auto':
>> +        symmetric_cbar = stat_map_min < 0
>
>To be frank I don't think I have encountered such a use case in the
>past: while positive values van be seen for variance images, I have no
>idea of the equivalent for negative images.
>On the other hand, I thinks that adding this use case could be useful
>in some cases that I can't figure out at the moment, and will not harm.
>
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/nilearn/nilearn/pull/293/files#r22573993